pyproject.toml

```diff
@@ -115,6 +115,7 @@ optional = [
     # "openbabel>=3.1.1; platform_system=='Linux'",
 ]
 numba = ["numba>=0.55"]
+numpy1 = ["numpy>=1.25.0,<2"]  # Test NP1 on Windows (quite buggy ATM)
```

Comment:
nice, just to check did you look at the CI dep install logs? just want to be sure it's actually installing `numpy<2`

Reply:
[Positive](https://github.com/materialsproject/pymatgen/actions/runs/10537208035/job/29198324593?pr=3992):
>  + numpy==1.26.4